src/Utilities.js

```diff
@@ -64,3 +66,23 @@ export const getLookupConfigItems = (propertyTemplate) => {
 
   return templateConfigItems
 }
+
+/**
+ * Loads N3 into a dataset.
```

Comment:
into an rdf.Dataset?  into an rdf-ext Dataset object?  And it's actually loading ttl as a string, no?

Reply:
an rdf-ext Dataset is an implementation of rdf.Dataset.